backends-velox/src/test/scala/io/glutenproject/execution/TestOperator.scala

```diff
@@ -1086,4 +1086,47 @@ class TestOperator extends VeloxWholeStageTransformerSuite with AdaptiveSparkPla
     }
   }
 
+  test("Support In list option contains non-foldable expression") {
```

Comment:
can you add this similar ut for the ch backend ?

Reply:
I try to add test for ch backend, let's see if the tests can pass..